bears/general/MementoBear.py

```diff
@@ -11,15 +13,27 @@
 from memento_client import MementoClient
 
 
+def get_archive_timegate_tuple(archive_registry_url):
+    resp = requests.get(archive_registry_url)
+    # Parse the xml file
+    data = etree.fromstring(resp.content)
+    timegate_list = [link for link in data.xpath('./link/timegate/@uri')]
+
+    return tuple(timegate_list)
+
+
 class MementoBear(LocalBear):
     DEFAULT_TIMEOUT = 15
     LANGUAGES = {'All'}
-    REQUIREMENTS = {PipRequirement('memento_client', '0.5.3')}
+    REQUIREMENTS = {PipRequirement('memento_client', '0.5.3'),
+                    PipRequirement('lxml', '3.6.0')}
```

Comment:
just a note, lxml is an optional dependency in https://github.com/mementoweb/py-memento-client/blob/master/setup.py#L90